tests/e2e/k8s_test.go

```diff
@@ -127,6 +138,7 @@ func createK8sResourcesFromManifest(kubectlOptions *k8s.KubectlOptions, manifest
 		GinkgoT(),
 		kubectlOptions,
 		"create",
+		"-n", kubectlOptions.Namespace,
```

Comment:
This should not be necessary. The -n namespace should be included automatically in the query.

Reply:
Just checked, not sure why I added that but I seemed to remember a kafka cluster wrongly deploying into the default namespace but it seems correct now, may have been something else, will remove.